notifiers/providers/dingtalk.py

```diff
@@ -0,0 +1,39 @@
+from ..core import Provider
+from ..core import Response
+from ..utils import requests
+
+
+class DingTalk(Provider):
+    """Send DingTalk notifications"""
+
+    base_url = "https://oapi.dingtalk.com/robot/send?access_token={}"
+    site_url = "https://oapi.dingtalk.com/"
+    path_to_errors = ("message",)
+    name = "dingtalk"
+
+    _required = {"required": ["access_token", "msg_data"]}
+    _schema = {
+        "type": "object",
+        "properties": {
+            "access_token": {
+                "type": "string",
+                "title": "access token to pair a channel to receive notification",
+            },
+            "msg_data": {
+                "type": "object",
+                "title": "dingtalk message body definition",
```

Comment:
you should elaborate on the required schema here, this is the interesting part :-)

Reply:
ok, i'll try.